lib/widgets/autocomplete.dart

```diff
@@ -165,7 +167,54 @@ class ComposeAutocomplete extends AutocompleteField<MentionAutocompleteQuery, Me
   @override
   MentionAutocompleteView initViewModel(BuildContext context) {
     final store = PerAccountStoreWidget.of(context);
-    return MentionAutocompleteView.init(store: store, narrow: narrow);
+    return MentionAutocompleteView.init(store: store, narrow: narrow, wildcards: _wildcards(context, store));
+  }
+
+  List<Wildcard> _wildcards(BuildContext context, PerAccountStore store) {
+    final isDmNarrow = narrow is DmNarrow;
+    final isChannelWildcardAvailable = store.account.zulipFeatureLevel >= 247; // TODO(server-9)
+    final isTopicWildcardAvailable = store.account.zulipFeatureLevel >= 188; // TODO(sever-8)
+    final zulipLocalizations = ZulipLocalizations.of(context);
+    return [
+      Wildcard(
+        name: zulipLocalizations.all,
+        value: 'all',
+        fullDisplayName: 'all (${isDmNarrow
+          ? zulipLocalizations.notifyRecipients
+          : zulipLocalizations.notifyChannel(isChannelWildcardAvailable
+              ? "channel" : "stream")})',
```

Comment:
I think it will be cleaner to move this `fullDisplayName` logic to where the result is actually being displayed, at build time.

Reply:
Also avoid string manipulation on translated strings. See the "general background" at the end of this section:
https://github.com/zulip/zulip-mobile/blob/main/docs/howto/translations.md#all-code-contributors-wiring-our-code-for-translations

Instead, put the two parts of this into separate widgets or `TextSpan`s. That way they can also get different text styles.